setup.py

```diff
@@ -21,7 +21,7 @@ def get_requirements():
 
 setup(
     name="pubtools-pulplib",
-    version="2.34.5",
+    version="2.36.0",
```

Comment:
Seems like this is being bumped more than necessary, perhaps intentional because there is another PR in progress which could potentially be released as 2.35.0 prior to this one?  Anyway, this should not be harmful, but if they are really submitted around the same time then one of the PRs is doomed to conflict.

Reply:
This version was suggested by pidiff I think even on my local machine